src/App.jsx

```diff
@@ -1,14 +1,18 @@
-import React, { useState, useEffect } from 'react';
-import './App.css';
+import { useState, useEffect } from "react";
+import "./App.css";
 
 function App() {
```

Comment:
I didn't want to change all the code but i'd definitively create a component named `<TodoList />`, so the `<App>` itself would be: 

```js
function App() {
  return <TodoList />
}
```

so later (exercise no.3) will be easier to render the Todo list or the Login form 

_guiño guiño ;) ;) ;) ;)_